base/terminal.py

```diff
@@ -33,18 +35,26 @@ def log(self, message, prefix=True):
         else:
             print(message)
 
-    def get_input(self, prompt=None):
+    def get_input(self, prompt=None, prefix=True):
         """Wrapper around input() to have a terminal like appearance"""
-        if prompt is not None:
-            self.log(f'{self.prefix} {prompt}')
-        return input(f'{self.prefix} ')
+        if prefix:
+            if prompt is not None:
+                self.log(f'{self.prefix} {prompt}')
+
+            return input(f'{self.prefix} ')
+        return input(f'{prompt}')
 
     def run(self):
         """Runs the terminal in loop"""
 
         latest_session = SessionDao.get_last_session()
         if latest_session is None:
             self.log('Welcome to Distributed Finder!', prefix=False)
+            # make the sector divisions
+            # should NOT be here
+            SectorDao.create_sectors_division(
```

Comment:
Does this check if sectors were already created?

Reply:
the function? nopes, doesnt. The only reason it should not be here is cause it might be messing with your design. warna it works flawlessly XD

Reply:
Oh, it creates the sectors only if there is no session. Got it!